ios/Framework/Core/Asset/ListScreenlet/AssetListPageLiferayConnector.swift

```diff
@@ -132,6 +132,20 @@ public class AssetListPageLiferayConnector: PaginationLiferayConnector {
 		return entryQuery
 	}
 
+
+	//MARK: Private methods
+
+	private func handleUserVisibleFlag(values: [String : AnyObject]) -> [String : AnyObject] {
+		if (classNameId == AssetClasses.getClassNameId(AssetClassNameKey_User)) {
+			var newValues = values
+
+			newValues.updateValue("false", forKey: "visible")
+			newValues.updateValue("", forKey: "groupIds")
```

Comment:
- Why not `newValues["visible"]` = false?
- Should `false` be a string?
- Why empty `groupIds`?